fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java

```diff
@@ -514,6 +516,12 @@ private void onFinished(OlapTable tbl) {
      */
     @Override
     protected boolean cancelImpl(String errMsg) {
+        //colocate mv
+        Database db = GlobalStateMgr.getCurrentState().getDb(dbId);
+        OlapTable table = (OlapTable) db.getTable(tableId);
+        db.writeLock();
+        table.removeMaterializedViewWhenJobCanceled(rollupIndexName);
```

Comment:
why use `removeMaterializedViewWhenJobCanceled` instead of `removeColocateMaterializedView`, suggest add some comments